api/src/main/resources/client_api.yaml

```diff
@@ -74,6 +74,70 @@ paths:
           schema:
             $ref: "#/definitions/MaterializeCohortResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/concepts:
+      post:
+        tags:
+          - concepts
+
+        description: >
+          Searches for concepts in concept table by name, and optionally filter on
+          domain, vocabulary ID, or standard concept status. Uses the CDR version affiliated
+          with the workspace specified in the path.
+
+        operationId: "searchConcepts"
+        parameters:
+          - in: path
+            name: workspaceNamespace
+            description: The namespace of the workspace for which concepts are being looked up.
+            required: true
+            type: string
+          - in: path
+            name: workspaceId
+            description: The ID of the workspace for which concepts are being looked up.
+            required: true
+            type: string
+          - in: body
+            name: query
+            schema:
+              type: string
+            required: true
+            description: >
+              A query string that can be used to match a subset of the name (case-insensitively),
+              the entire code value (case-insensitively), or the concept ID.
+          - in: body
+            name: standard_concept_filter
```

Comment:
Field names should be lower-camel-cased for consistency with the rest of the api

Reply:
Done.